exercises/parallel-letter-frequency/tests/parallel-letter-frequency.rs

```diff
@@ -124,3 +124,18 @@ fn test_all_three_anthems_3_workers() {
     assert_eq!(freqs.get(&'t'), Some(&56));
     assert_eq!(freqs.get(&'ü'), Some(&2));
 }
+
+#[test]
+#[ignore]
+fn test_non_integer_multiple_of_threads() {
+    let mut v = Vec::with_capacity(999);
+    for _ in 0..999 {
+        v.push("abc");
+    }
```

Comment:
I would expect clippy to request `.resize` or static creation here.

Reply:
Oops!  Trying to split up my three PRs didn't go as well!